src/redux/services/compare/filterResource.ts

```diff
@@ -0,0 +1,24 @@
+import {ResourceComparison} from '@redux/reducers/compare';
+
+type FilterOptions = {
+  search?: string;
+};
+
+type ResourceFilter = (value: ResourceComparison) => boolean;
```

Comment:
would it make sense to rename this to `ResourceSetFilter` ? just to not mistakenly use the other `ResourceFilter` type that we have in the project

Reply:
Oh yeah, I mistakenly used K8sResource in the beginning and forgot to rename this boy. I'll update it to `ComparisonFilter`.